src/site/content/en/blog/optimize-vitals-lighthouse/index.md

```diff
@@ -0,0 +1,369 @@
+
+For post-load CLS, I personally find a lot of value in _persistently_ visualizing with rectangles
+which elements contributed the most to CLS. This is a feature you'll find in third-party tools like
+SpeedCurve's [Core Web Vitals dashboard](https://speedcurve.com/blog/web-vitals-user-experience/)
+and which I love using [Defaced's Layout Shift GIF
+Generator](https://defaced.dev/tools/layout-shift-gif-generator/) for:
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/ju6XjKBYzF6G537myjUW.gif", alt="the layout shift generator highlighting shifts", width="800", height="450" %}
+</figure>
+
+For a site-wide view of layout shift issues, I get a lot of mileage out of [Search Console's Core
+Web Vitals report](https://support.google.com/webmasters/answer/9205520?hl=en). This lets me see
+the kinds of pages on my site with a high CLS (in this case helping self-identify what template
+partials I need to spend my time on):
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/2Ihb2GYkbpGzYLYoZEDP.png", alt="Search Console displaying CLS issues", width="800", height="506" %}
+</figure>
+
+### Identifying CLS from images without dimensions
+
+To [limit](/optimize-cls/#images-without-dimensions) Cumulative Layout Shift being caused by images
+without dimensions, include width and height size attributes on your images and video elements.
+This approach ensures that the browser can allocate the correct amount of space in the document
+while the image is loading.
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/fZRkmM18rvfy6y7LB1Qx.png", alt="Audit for image elements without explicit width and height", width="800", height="489" %}
+</figure>
+
+See [Setting Height And Width On Images Is Important
+Again](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/) for a
+good write-up on the importance of thinking about image dimensions and aspect ratio.
+
+### Identifying CLS from advertisements
+
+[Publisher Ads for Lighthouse](https://developers.google.com/publisher-ads-audits) allows you to
+find opportunities to improve the loading experience of ads on your page, including contributions
+to layout shift and long tasks that may push out how soon your page is usable by users. In
+Lighthouse, you can enable this via Community Plugins.
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/kR3jgctso6Hg0OxD8xwi.png", alt="Ads related audits highlighting opportunities to reduce time to request and layout shift", width="800", height="527" %}
+</figure>
+
+Remember that ads are one of the
+[largest](/optimize-cls/#ads-embeds-and-iframes-without-dimensions) contributors to layout shifts
+on the web. It's important to:
+
++   Take care when placing non-sticky ads near the top of the viewport 
++   Eliminate shifts by reserving the largest possible size for the ad slot
+
+### Preload optional fonts to prevent layout shifts
```

Comment:
Would your preference be that I drop the "preload optional fonts to prevent layout shifts" section in light of that? or strengthen the caveats? I'd be comfortable removing and originally excluded this audit for similar concerns.

Reply:
SGTM. Removed in a follow-up commit :)